src/Components/test/E2ETest/Tests/WebAssemblyGlobalizationTest.cs

```diff
@@ -10,11 +10,13 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests
+namespace Microsoft.AspNetCore.Components.E2ETest.Tests
 {
     // For now this is limited to server-side execution because we don't have the ability to set the
     // culture in client-side Blazor.
-    public class WebAssemblyGlobalizationTest : GlobalizationTest<ToggleExecutionModeServerFixture<Program>>
+    // This type is internal since localization currently does not work.
+    // Make it public onc https://github.com/dotnet/runtime/issues/38124 is resolved.
```

Comment:
Is there an issue tracking the need for us to do this? If not we might not remember to re-enable this test even when https://github.com/dotnet/runtime/issues/38124 is resolved